### PR TITLE
Analytics

### LINKED ISSUE #92: Analytics
Opt-out usage analytics

### PR DESCRIPTION
**Pull request by tomusdrw**
Closes #92

### CODE REVIEW

File: background/analytics.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    // initialize
+    analytics.initialize(trackingId, {
+      debug: !isProd,
+      clientId: config.clientId
+    });
+    // Disable protocol check
+    analytics.set('checkProtocolTask', null);
+    analytics.set('dimension1', config.lastVersion);
+    analytics.set('dimension2', config.lastChain);
```
Comment (ngotchac) [inline background/analytics.js]: Why `dimension1` and `2`, instead of more explicit names?

File: background/transport.js [hunk 2]
```diff
@@ -236,16 +237,34 @@ export default class Transport {
...
+      ])
+        .then(([version, chain]) => {
+          Config.set({
+            lastVersion: version,
+            lastChain: chain
+          });
+
+          analytics.ifEnabled(() => {
+            analytics.set('dimension1', version);
+            analytics.set('dimension2', chain);
```
Comment (ngotchac) [inline background/transport.js]: Same comment as above. Also, could use variables for the keys.

**Comment by ngotchac** [comment]
A few questions, but otherwise LGTM !
(Although I was wondering if we should ask the user on installation/update of the extension if (s)he wants to opt in or out...)

File: background/analytics.js (hunk 1 above)
Comment (tomusdrw) [reply background/analytics.js]: It's required by analytics platform afaict: https://support.google.com/analytics/answer/2709828?hl=en#collection

More explicit names can be defined in the analytics UI.

File: background/transport.js (hunk 2 above)
Comment (tomusdrw) [reply background/transport.js]: Constants are a good idea, will change that and give the constants more meaningful names with a comment why the values are so weird. Thanks!

File: background/analytics.js (hunk 1 above)
Comment (ngotchac) [reply background/analytics.js]: Ah yes right, I remember those...

**Comment by tomusdrw** [comment]
@ngotchac I think it deserves a separate issue and PR. Asking the question explicitly is more like opting-in (and imho would be better to keep it as opt-out), but I think it would be good to open a welcome page where:
1. You have a link to download Parity (especially that we want to make the extension public)
2. You have details about anayltics collection and information that it can be disabled via extension options.

**Review by ngotchac — APPROVED** [review]

**Pull request merged**